src/ServiceV2/NotificationService.php

```diff
@@ -71,6 +73,14 @@ public function sendSmsReminder(Rappel $reminder): void
             $message = $this->getReminderMessage($event);
             $this->sendSms($number, $message);
             $reminder->setBEnvoye(true);
+
+            $sms = (new SMS())
+            ->setRappel($reminder)
+            ->setEvenement($event)
+            ->setBeneficiaire($beneficiary)
+            ->setDest($number);
```

Comment:
$sms = SMS::CreateFromEvent($event)

Reply:
just for readability, I would also have written a method inside service `$this->createSmsForEvent($event)` or `persistEventSms`

Reply:
This call may even live inside $this->sendSms no ?